sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/AuthenticationsImpl.java

```diff
@@ -25,103 +23,115 @@
 import com.azure.core.util.FluxUtil;
 import reactor.core.publisher.Mono;
 
-/** An instance of this class provides access to all the operations defined in AccessTokens. */
-public final class AccessTokensImpl {
+/** An instance of this class provides access to all the operations defined in Authentications. */
+public final class AuthenticationsImpl {
     /** The proxy service used to perform REST calls. */
-    private final AccessTokensService service;
+    private final AuthenticationsService service;
 
     /** The service client containing this operation class. */
     private final ContainerRegistryImpl client;
 
     /**
-     * Initializes an instance of AccessTokensImpl.
+     * Initializes an instance of AuthenticationsImpl.
      *
      * @param client the instance of the service client containing this operation class.
      */
-    AccessTokensImpl(ContainerRegistryImpl client) {
+    AuthenticationsImpl(ContainerRegistryImpl client) {
         this.service =
-                RestProxy.create(AccessTokensService.class, client.getHttpPipeline(), client.getSerializerAdapter());
+                RestProxy.create(AuthenticationsService.class, client.getHttpPipeline(), client.getSerializerAdapter());
         this.client = client;
     }
 
     /**
-     * The interface defining all the services for ContainerRegistryAccessTokens to be used by the proxy service to
+     * The interface defining all the services for ContainerRegistryAuthentications to be used by the proxy service to
      * perform REST calls.
      */
-    @Host("{endpoint}")
-    @ServiceInterface(name = "ContainerRegistryAcc")
-    public interface AccessTokensService {
+    @Host("{url}")
+    @ServiceInterface(name = "ContainerRegistryAut")
+    public interface AuthenticationsService {
         // @Multipart not supported by RestProxy
-        @Post("/oauth2/token")
+        @Post("/oauth2/exchange")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(AcrErrorsException.class)
-        Mono<Response<AccessToken>> get(
-                @HostParam("endpoint") String endpoint,
-                @BodyParam("application/x-www-form-urlencoded")
-                        PathsV3R3RxOauth2TokenPostRequestbodyContentApplicationXWwwFormUrlencodedSchema refreshToken,
+        Mono<Response<AcrRefreshToken>> exchangeAadAccessTokenForAcrRefreshToken(
+                @HostParam("url") String url,
+                @BodyParam("application/x-www-form-urlencoded") String grantType,
+                @BodyParam("application/x-www-form-urlencoded") String service,
+                @BodyParam("application/x-www-form-urlencoded") String accessToken,
```

Comment:
Is this a code generation change?

@jianghaolu, @srnagar

Reply:
This could be due to a change in swagger. Nothing related to urlencoded content type has changed in autorest afaik.

Reply:
Should we update `SwaggerMethod` in Core to enable support for multiple `BodyParam` parameters? Right now it only handles having one.

Reply:
I'd also expect that `FormParam` would be used here.